### PR TITLE
feat(roles): display role names in members list

### PR DESCRIPTION
**Pull request by cathteng**
The role names on the members list page should show all the rows a member has. The filter should take all roles into account too.

Screenshot 2023-03-31 at 13 52 56

For ER-1358

### CODE REVIEW

File: static/app/views/settings/organizationMembers/organizationMemberRow.tsx [hunk 1]
```diff
@@ -75,20 +78,33 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
...
   renderMemberRole() {
-    const {member} = this.props;
-    const {roleName, pending, expired} = member;
+    const {member, organization} = this.props;
+    const {orgRole, pending, expired, orgRolesFromTeams} = member;
+    const {orgRoleList} = organization;
+
+    const allOrgRoleNames =
+      orgRolesFromTeams?.filter(obj => obj.role !== null).map(obj => obj.role.id) ?? [];
+    allOrgRoleNames?.push(orgRole);
```
Comment (evanpurkhiser) [inline static/app/views/settings/organizationMembers/organizationMemberRow.tsx:87]: Extraneous `?.`
```suggestion
    allOrgRoleNames?push(orgRole);
```

**Comment by evanpurkhiser** [comment]
A better PR title I think would be

feat(roles): Display role names in members list

**Comment by leedongwei** [comment]
This feature has a lot of surface area and it can get confusing. 

We should have a common component to explain it: https://github.com/getsentry/sentry/pull/46355

File: fixtures/js-stubs/member.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ export function Member(params = {}) {
     },
     user: User(),
     inviteStatus: 'approved',
+    orgRolesFromTeams: [],
```
Comment (leedongwei) [inline fixtures/js-stubs/member.js:21]: Can you move this higher, so it's next to `orgRole`

**Review by leedongwei — APPROVED** [review]

**Pull request merged**